aspnetcore/fundamentals/environments.md

```diff
@@ -69,6 +69,48 @@ The `Production` environment is the environment in which the application runs wh
 
 This is by no means meant to be a complete list. It's best to avoid scattering environment checks in many parts of your application. Instead, the recommended approach is to perform such checks within the application's `Startup` class(es) wherever possible
 
+## Setting the environment using the command line
```

Comment:
Drop "using the command line"  since some of the methods involve editing files rather than command line